src/saka/Main/Components/GUIContainer/index.jsx

```diff
@@ -25,6 +27,15 @@ export default class GUIContainer extends Component {
     this.setState({ zoom });
   };
 
+  fetchTheme = async function fetchTheme() {
+    const storedSettings = ( await browser.storage.sync.get(['sakaSettings']) ) || {};
+    console.log(storedSettings);
```

Comment:
Once we are ready to merge this should be deleted

Reply:
again, was trying to figure stuff out at this point

- I haven't written a Firefox addon before
- I haven't dealt with (P)React before

when stuff wasn't working, I fell back on old-school debugging, only to find that the problem was that since I'd had to restart Firefox, it had unloaded my dev variant of the plugin and loaded back the original version :woman_facepalming: